index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+  <link rel="stylesheet" href="style.css">
+</head>
+  
+  <body>
+
+    <header>
+      <div class = "banner">
+        <div class = "banner_button">
+          <a class = "logo" href="/"><img src = "image/logo.png"></a>
```

Comment:
❗️ 수정요청
불필요한 공백은 없는것이 좋습니다~

```suggestion
          <a class="logo" href="/"><img src="image/logo.png"></a>
```